src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -417,8 +417,33 @@ class LiteralCoercionSegment(BaseSegment):
     )
 
 
+@bigquery_dialect.segment(replace=True)
+class ObjectReferenceSegment(ansi_dialect.get_segment("ObjectReferenceSegment")):  # type: ignore
```

Comment:
not sure I understand why you need to use the ansi_dialect segment as a base and then not overwrite anything?

Reply:
Can't you just import the ObjectReferenceSegment from the ansi_dialect package as a class?

Reply:
I'll take a look at that. I'm in unfamiliar territory here -- replacing a subclass of `ObjectReferenceSegment` in a child dialect (BigQuery). 🤯

Reply:
Done! IIUC, Alan did some refactoring a few months back, and dialects shouldn't directly `import` from other dialects, so I did this:
```
ObjectReferenceSegment = ansi_dialect.get_segment("ObjectReferenceSegment")
```

This seems to work fine -- I can use `ObjectReferenceSegment` as a base class, which was why I had created this unnecessary new class. 👍